src/Files.App.Controls/ThemedIcon/Styles/Icons.Action.xaml

```diff
@@ -0,0 +1,223 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Files.App.Controls">
+
+	<Style
+		x:Key="App.ThemedIcons.Action_Copying"
```

Comment:
Can `Actions` be removed or alternatively, the underscore be changed to a period?

Reply:
Done